src/content/docs/release-notes/mobile-release-notes/android-release-notes/android-ndk-110.mdx

```diff
@@ -0,0 +1,19 @@
+---
+subject: Android agent
+title: Android NDK agent v1.1.0
+releaseDate: '2024-06-27'
+version: 1.1.0
+---
+
+## New
+
+* Reduce the overall size of the agent.
+* Extend AgentNDK.Builder.withANRMonitor(boolean) to disable native ANR detection as well.
```

Comment:
@ywang-nr is there a way for us mark this up so it is more readable? For example, could we put it inside back ticks or something: `gentNDK.Builder.withANRMonitor(boolean)`

Is the (boolean) part of the expression?

Reply:
Yes, (boolean is part of the expression), the suggested changes are good.